apluslms-yamlidator/apluslms_yamlidator/utils/collections.py

```diff
@@ -93,44 +93,51 @@ def data_updated(self, key, data):
     def get_data(self):
         return self._data
 
+    @property
+    def top_parent(self):
```

Comment:
This is typically called `root`. As `_parent` is non-public, this could be too.

Reply:
Ah yes thank you, I had trouble figuring a good name :p